<launch>
    <include file="$(find hardware_interface)/launch/sentry_serial.launch">
    </include>

    <include file="$(find fdilink_ahrs)/launch/ahrs_data.launch">

    </include>

    <node pkg="sentry_navigation" type="chassis_odom" name="chassis_odom"/>

    <include file="$(find robot_pose_ekf)/robot_pose_ekf.launch">

    </include>

</launch>
